### PR TITLE
Add CCM e2e AWS testing job for WMCO

### PR DESCRIPTION
**Pull request by Fedosin**
This PR adds an e2e job:

* aws-e2e-ccm-install tests fresh installations with CCM support.

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -69,6 +69,20 @@ tests:
...
     from: src
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: aws-e2e-ccm
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    test:
+    - ref: windows-e2e-operator-test
+    workflow: openshift-e2e-aws-ccm-ovn-hybrid
+- as: aws-e2e-ccm-install
```
Comment (sebsoto) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:79]: Is this job going to be a temporary addition?
CCM will be running in the other jobs at some point, correct?

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 2]
```diff
@@ -69,6 +69,20 @@ tests:
   container:
     from: src
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: aws-e2e-ccm
```
Comment (sebsoto) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:72]: `tests migration from in-tree cloud provider to CCM on a running cluster`
I'm not sure if this job is going to be helpful, as Windows Nodes are added only after the cluster is fully up and running, as part of the `windows-e2e-operator-test` pod.

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml (hunk 1 above)
Comment (Fedosin) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:79]: Yes, that's right. In-tree cloud providers will be removed soon, and CCM will be the only supported mechanism.

Comment (sebsoto) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:72]: This test should also be added to the WMCB repo

**Review by sebsoto — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 3]
```diff
@@ -69,6 +69,12 @@ tests:
   container:
     from: src
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: aws-e2e-ccm-install
```
Comment (mansikulkarni96) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:72]: No harm in adding skip_if_only_changed field same as other jobs, would be useful when the job is functional.

**Review by mansikulkarni96 — COMMENTED** [review]
Thanks for adding this @Fedosin!

**Review by elmiko — COMMENTED** [review]
+1, this generally looks good to me but i don't know the wmco infra that well.

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml (hunk 3 above)
Comment (Fedosin) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:72]: Done!

**Pull request merged**